### PR TITLE
Remove bootstrap from admin/broadcasts and use Crayons

### PR DESCRIPTION
**Pull request by rt4914**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR is a step towards removing bootstrap completely and replacing it with crayons.

The changes are only design related for which all before vs after screenshots have been shown below.

## Related Tickets & Documents

- Related Issue #18756 
- Closes #

### Video which displays all major changes
- [Loom](https://www.loom.com/share/1ffc65ad66ff4b79b43551f8e088baea)

### Before vs After -- Broadcast

| Before  | After |
| ------------- | ------------- |
| Screenshot 2022-12-14 at 4 54 38 PM | Screenshot 2022-12-14 at 4 57 59 PM  |
| Screenshot 2022-12-14 at 4 54 55 PM | Screenshot 2022-12-14 at 4 58 29 PM  |
| Screenshot 2022-12-14 at 4 55 06 PM  | Screenshot 2022-12-14 at 4 58 47 PM  |
| Screenshot 2022-12-14 at 4 55 15 PM  | Screenshot 2022-12-14 at 4 58 53 PM  |
| Screenshot 2022-12-14 at 4 55 38 PM | Screenshot 2022-12-14 at 5 01 04 PM  |
|     | Screenshot 2022-12-14 at 5 01 15 PM  |
| Screenshot 2022-12-14 at 4 55 46 PM | Screenshot 2022-12-14 at 5 01 31 PM  |
| Screenshot 2022-12-14 at 4 55 53 PM | Screenshot 2022-12-14 at 5 01 38 PM |
|   | Screenshot 2022-12-14 at 5 01 45 PM |
| Screenshot 2022-12-14 at 4 56 02 PM  | Screenshot 2022-12-14 at 5 01 59 PM |
| Screenshot 2022-12-14 at 4 56 09 PM  | Screenshot 2022-12-14 at 5 02 13 PM |

### UI accessibility concerns?

No

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

Just keep an eye on the admin/broadcasts section to make sure UI does not break.

### CODE REVIEW

**Comment by rt4914** [comment]
@anujbhavsar96 PTAL at before/after screenshots. There are minor changes in design and those changes have been done keeping in mind that they look similar to other screens that we have in admin panel.

**Review by anujbhavsar96 — APPROVED** [review]

**Comment by rt4914** [comment]
@jaw6 @Ridhwana This PR is ready for review.

**Review by jaw6 — APPROVED** [review]
Code changes look fine to me -- the small visual changes will take a little getting used to, but seem fine? 👍

**Comment by rt4914** [comment]
@Ridhwana Waiting for your PR on this and after that I will merge it.

**Comment by rt4914** [comment]
@Ridhwana Will wait for your review on this and merge afterwards.

File: app/views/admin/broadcasts/index.html.erb
Comment (Ridhwana) [inline app/views/admin/broadcasts/index.html.erb:12]: Question: @rt4914 is there a reason we change from `link_to` to an `a` tag in this file?

**Review by Ridhwana — APPROVED** [review]
LGTM, left a question regarding the use of link_to.

**Comment by Ridhwana** [comment]
> @Ridhwana Waiting for your review on this and after that I will merge it.

Sorry I missed this, I was on PTO on Friday.

File: app/views/admin/broadcasts/index.html.erb
Comment (rt4914) [reply app/views/admin/broadcasts/index.html.erb:12]: The only reason was keeping it consistent with what Pawel did in https://github.com/forem/forem/pull/16359 PR in `app/views/admin/articles/index.html.erb`.

Should I revert it back to using `link_to` ? Also, what is the major difference between the two?

Comment (Ridhwana) [reply app/views/admin/broadcasts/index.html.erb:12]: In general, it's usually more efficient when having complex paths and saves us from messy interpolations with html and erb. However, in this scenario there's no huge difference because the path is a straightforward one, however for consistency purposes I'm of the opinion that it makes more sense to use the `link_to` helper.

Comment (rt4914) [reply app/views/admin/broadcasts/index.html.erb]: Thanks for the clarification. I have used `link_to` instead of `a` tag now.

**Pull request merged**
